plugins/obs-webrtc/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+project(obs-webrtc)
+
+include(FetchContent)
+
+FetchContent_Declare(
+  Corrosion
+  GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git
+  GIT_TAG v0.3.1)
+
+FetchContent_MakeAvailable(Corrosion)
+
+add_library(obs-webrtc MODULE)
+
+if(OS_WINDOWS)
+  set(MODULE_DESCRIPTION "OBS webrtc module")
+  configure_file("${CMAKE_SOURCE_DIR}/cmake/bundle/windows/obs-module.rc.in"
+                 obs-webrtc.rc)
+  list(APPEND obs-webrtc_SOURCES obs-webrtc.rc)
+endif()
+
+corrosion_import_crate(MANIFEST_PATH webrtc/Cargo.toml)
+corrosion_set_env_vars(webrtc-rs-lib
+                       "OBS_WEBRTC_GENERATED_DIR=${CMAKE_CURRENT_BINARY_DIR}")
+
+target_sources(obs-webrtc PRIVATE whip-service.c whip-output.c webrtc.c
+                                  whip-output.h)
+target_link_libraries(obs-webrtc OBS::libobs webrtc-rs-lib)
+target_include_directories(obs-webrtc PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Is this required for the installed Cargo modules or just the `.rc` file on Windows?

Reply:
Which line specifically are you referring to? The include directory is where the generated header is plopped. The rc file is, admittedly, copied from the previous PR so I'll make sure I match how others manage that file!

Reply:
Yeah I don't see any header being generated here. Or is that implicitly done by `corrosion`?

In the future we'll have the binary dir added by default anyway, so this is not "wrong", it's currently only added if needed, so it's just a semantical issue. But should be fine then.

Reply:
`./build/plugins/obs-webrtc/bindings.h` It should be emitted when the build.rs runs during a build

Reply:
`bindings.h` doesn't appear in the `target_sources` however, you might want to add that. It's then obvious why the build directory is needed as an include directory.

(Header files per-se do not need to be added to a project in CMake, but it's good practice to do so, as they'll appear in Xcode/Visual Studio then).

Reply:
@PatTheMav Added generated bit to header and put it in target_sources, does that seem reasonable?

Reply:
Huh, I only see `target_sources(obs-webrtc PRIVATE whip-service.c whip-output.c webrtc.c whip-output.h)`, but no `bindings.h` - is GitHub being weird maybe?

Reply:
I think it's showing the outdated one, as well as the other comment